drghs-worker/maintnerd/main.go

```diff
@@ -228,10 +229,64 @@ func main() {
 		log.Printf("internal gRPC server listening on: %s", *intListen)
 		return grpcServer.Serve(lis)
 	})
+
+	group.Go(
+		// Get SLO rules for the repo
+		func() error {
+			parent := fmt.Sprintf("owners/%s/repositories/%s", *owner, *repo)
+
+			ticker := time.NewTicker(10 * time.Minute)
+
+			for t := range ticker.C {
+				log.Printf("Slo sync at %v", t)
+
+				_, err := getSlos(ctx, parent)
+				if err != nil {
+					logAndPrintError(err)
+					log.Printf("Slo sync err: %v", err)
+				}
+			}
+			return nil
+		})
+
 	err = group.Wait()
 	log.Fatal(err)
 }
 
+func getSlos(ctx context.Context, parent string) ([]*drghs_v1.SLO, error) {
+	dialCtx, cancel := context.WithTimeout(ctx, time.Second)
+	defer cancel()
+
+	conn, err := grpc.DialContext(dialCtx, *sloAddress, grpc.WithInsecure(), grpc.WithBlock())
+	if err != nil {
+		return nil, fmt.Errorf("Error connecting to SLO server: %v", err)
+	}
+	defer conn.Close()
```

Comment:
nit: consider looking at maintner-rtr or samplr-rtr to use the grpc-interceptors for automatic retry

Reply:
Do we want to add tracing with `grpctrace` as well?

Reply:
That would be great but not necessary.